app/server/lib/DocApi.ts

```diff
@@ -595,6 +603,29 @@ export class DocWorkerApi {
       archive.dataStream.pipe(res);
     }));
 
+    this._app.post('/api/docs/:docId/attachments/archive', isOwner, withDoc(async (activeDoc, req, res) => {
```

Comment:
Is `isOwner` the correct permission level for this endpoint?

Reply:
I don't think so. At the overall level, being a viewer should suffice. There should be an internal check that the user does have full read rights to all of the doc though - like ability to download / copy all.

`isOwner` here would mean that when we go to implement copy, the users who can copy a document with attachments properly would be limited to owners IFF those attachments are external, which would be hard to explain.

Reply:
^ ignore all that, thought I was looking at get but this is clearly post - isOwner is fine!